### PR TITLE
metrics: accept any viable target update

### PR DESCRIPTION
**Pull request by pared**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

Related to https://github.com/iterative/dvc/pull/4590

### CODE REVIEW

File: content/docs/command-reference/metrics/diff.md [hunk 1]
```diff
@@ -56,6 +56,9 @@ lists all the current metrics without comparisons.
   $ dvc metrics diff HEAD v1 --targets t1.json t2.json
   ```
 
+  Note, that when using `--targets` it is possible to `diff` files that are not
+  specifically marked as metrics.
+
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/diff.md:61]: We can avoid this new paragraph by changing the beginning of the --targets option desc from

`` `--targets <paths>` - limit command scope to these metric files. Using -R``
to something like
`` `--targets <paths>` - limit command scope to these metrics files (supports any file, even when not marked as metrics specifically). Using `-R` ``

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/diff.md:61]: (Notice I also fixed the -R at the end, which should be in back quotes.)

File: content/docs/command-reference/metrics/show.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ Finds and prints all metrics in the <abbr>project</abbr> by examining all of its
 > This kind of metrics can be defined with the `-m` (`--metrics`) and `-M`
 > (`--metrics-no-cache`) options of `dvc run`.
 
-If `targets` are provided, it will show those specific metric files instead.
-With the `-a` or `-T` options, this command shows the different metrics values
-across all Git branches or tags, respectively. With the `-R` option, some of the
-target can even be directories, so that DVC recursively shows all metric files
-inside.
+If `targets` are provided, it will show those specific files instead, even if
+they are not marked as `metric` type in `dvc.yaml`. With the `-a` or `-T`
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/show.md:26]:
```suggestion
they are not marked `metrics` in `dvc.yaml`. With the `-a` or `-T`
```

**Review by jorgeorpinel — CHANGES_REQUESTED** [review]
Thanks! Left some comments ☝️ 

If you're unable to further edit this soon, please reopen from a branch directly on this repo instead of a fork so it's easier to take it over.

**Comment by efiop** [comment]
@pared Let's not forget to update `plots` docs as well, since the changes are so related.

File: content/docs/command-reference/metrics/diff.md (hunk 1 above)
Comment (pared) [reply content/docs/command-reference/metrics/diff.md:61]: And problably we should include that info in `dvc metrics show --help`

**Comment by pared** [comment]
Closing in favor of #1809 
Sorry @jorgeorpinel, keep forgetting to create PR's from origin.

**Comment by jorgeorpinel** [comment]
Yeah no problem @pared wither way it works but if you open from origin it's easier for me to take it over if needed.

**Pull request closed without merging**